src/common/rollbar.config.js

```diff
@@ -0,0 +1,99 @@
+import { rollbarAccessToken } from 'common/app.constants';
+import rollbar from 'rollbar-browser';
+import stacktrace from 'stacktrace-js';
+import map from 'lodash/map';
+import defaults from 'lodash/defaults';
+import get from 'lodash/get';
+
+/* @ngInject */
+function rollbarConfig(envServiceProvider, $provide) {
+  let rollbarConfig = {
+    accessToken: rollbarAccessToken,
+    captureUncaught: true,
+    captureUnhandledRejections: true,
+    environment: envServiceProvider.get(),
+    enabled: !envServiceProvider.is('development'), // Disable rollbar in development environment
+    transform: transformRollbarPayload
+  };
+  let Rollbar = rollbar.init(rollbarConfig);
+
+  /* @ngInject */
+  $provide.decorator('$log', ($delegate) => {
+    // Add rollbar functionality to each $log method
+    angular.forEach(['log', 'debug', 'info', 'warn', 'error'], (ngLogLevel) => {
+      let rollbarLogLevel = ngLogLevel === 'warn' ? 'warning' : ngLogLevel;
+
+      let originalFunction = $delegate[ngLogLevel]; // Call below to keep angular $log functionality
+
+      $delegate[ngLogLevel] = (...args) => {
```

Comment:
Gets the args as a real array. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters